### PR TITLE
Add option --notty to spacewalk-ssh-push-init

### PR DESCRIPTION
**Pull request by mbrookhuis**
## What does this PR change?

When using a non-root user to perform spacewalk-ssh-push-init and setting sudo according to the documentation, running the spacewalk-ssh-push-init with the --register option, the bootstrap script will not work. It seems that the script used hangs. The root cause is that the bootstrap scripts, or rather sudo, will ask for the root password. The option --notty will add the ssh option -t. This will then give the administrator the possibility to enter the password.  

## GUI diff

No difference.

Before:

After:

- [x] **DONE**

## Documentation
- (OPTIONAL) [Mbrookhuis ssh push init 2 #622](https://github.com/uyuni-project/uyuni-docs/pulls)

- [x] **DONE**

## Test coverage
- No tests: Testing done in my environment. And there is no test case for this tool.

- [x] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: spacewalk/certs-tools/spacewalk-certs-tools.changes [hunk 1]
```diff
@@ -1,3 +1,7 @@
+-------------------------------------------------------------------
+Fri Oct 19 16:45:00 CEST 2020 - michael.brookhuis@suse.com
```
Comment (cbosdo) [inline spacewalk/certs-tools/spacewalk-certs-tools.changes:2]: Remove these two header lines: they will be added by Julio in at release time

**Review by cbosdo — CHANGES_REQUESTED** [review]
I have no skill to check the code change, just the format of the PR

**Review by cbosdo — APPROVED** [review]
LGTM

File: spacewalk/certs-tools/spacewalk-certs-tools.changes (hunk 1 above)
Comment (mbrookhuis) [reply spacewalk/certs-tools/spacewalk-certs-tools.changes]: Done

**Pull request merged**